src/router.ts

```diff
@@ -92,6 +92,7 @@ export const routes = [
       status: FfaListingStatus.listed,
       listingHash: route.params.listingHash,
       requiresWeb3: true,
+      canPurchase: false,
```

Comment:
this name strikes me as a little confusing and dangerous.  how about something like `enablePurchaseButton`?